From 8218c86bee781899a5de67c822257664b4eecd7d Mon Sep 17 00:00:00 2001
From: Shane Francis <bigbeeshane@gmail.com>
Date: Sat, 22 May 2021 22:41:16 +0100
Subject: [PATCH 1/2] Add kernel check

---
 nss_core.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/nss_core.c b/nss_core.c
index 758004c..4e8badb 100644
--- a/nss_core.c
+++ b/nss_core.c
@@ -52,7 +52,8 @@
 (((LINUX_VERSION_CODE >= KERNEL_VERSION(3, 10, 0)) && (LINUX_VERSION_CODE < KERNEL_VERSION(3, 11, 0)))) || \
 (((LINUX_VERSION_CODE >= KERNEL_VERSION(3, 18, 0)) && (LINUX_VERSION_CODE < KERNEL_VERSION(3, 19, 0)))) || \
 (((LINUX_VERSION_CODE >= KERNEL_VERSION(4, 4, 0)) && (LINUX_VERSION_CODE < KERNEL_VERSION(4, 5, 0)))) || \
-(((LINUX_VERSION_CODE >= KERNEL_VERSION(5, 4, 0)) && (LINUX_VERSION_CODE < KERNEL_VERSION(5, 5, 0))))))
+(((LINUX_VERSION_CODE >= KERNEL_VERSION(5, 4, 0)) && (LINUX_VERSION_CODE < KERNEL_VERSION(5, 5, 0)))) || \
+(((LINUX_VERSION_CODE >= KERNEL_VERSION(5, 10, 0)) && (LINUX_VERSION_CODE < KERNEL_VERSION(5, 11, 0))))))
 #error "Check skb recycle code in this file to match Linux version"
 #endif
 
-- 
2.31.1

